spt/features/bms_flashlight_fix.cpp

```diff
@@ -0,0 +1,157 @@
+#include "stdafx.h"
+#include "..\feature.hpp"
+#include "convar.hpp"
+
+#ifdef BMS
+
+static void BMSFlashLightFixCVarCallback(IConVar* pConVar, const char* pOldValue, float flOldValue);
+
+ConVar y_spt_bms_flashlight_fix("y_spt_bms_0point9_flashlight_fix",
+                                "0",
+                                0,
+                                "Disables all extra movement on flashlight. (0.9 ONLY, FOR NOW!!!)",
+                                BMSFlashLightFixCVarCallback);
```

Comment:
Warnings about this command not working on other versions are redundant as versions where it doesnt work the command should get deleted.